### PR TITLE
Issue_#1.Added webhook payload processing when commit

### PR DESCRIPTION
**Pull request by andrycreator**
Currently I am using Map as container for payload, but I think it makes sense to create truncated models that will match our needs to avoid numerous class casts. @yuriytkach what is you view on this?

### CODE REVIEW

File: src/main/java/com/ahaidychuk/dependencyversionupdater/model/WebHookPayload.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import java.util.List;
+
+/**
+ * Truncated payload of Git Push event for repository.
+ */
+@Data
+public class WebHookPayload {
+
+  private String after;   // commit Hash
+  private List<Commit> commits;
```
Comment (yuriytkach) [inline src/main/java/com/ahaidychuk/dependencyversionupdater/model/WebHookPayload.java:14]: Finals on method params is not bad, but how about having immutable model classes?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+server.port=3333
```
Comment (yuriytkach) [inline src/main/resources/application.properties]: why do you want to set it? If you need for local testing you can set it for `local` profile only

File: src/test/java/com/ahaidychuk/dependencyversionupdater/service/RepoServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest()
```
Comment (yuriytkach) [inline src/test/java/com/ahaidychuk/dependencyversionupdater/service/RepoServiceTest.java]: do you really want to make it SpringBootTest that will start the whole context? Do you really need the whole context here?

File: src/main/java/com/ahaidychuk/dependencyversionupdater/model/WebHookPayload.java (hunk 1 above)
Comment (andrycreator) [reply src/main/java/com/ahaidychuk/dependencyversionupdater/model/WebHookPayload.java:14]: Marked WebHook and Commit as final classes

File: src/main/resources/application.properties (hunk 2 above)
Comment (andrycreator) [reply src/main/resources/application.properties]: Fixed

File: src/test/java/com/ahaidychuk/dependencyversionupdater/service/RepoServiceTest.java (hunk 3 above)
Comment (andrycreator) [reply src/test/java/com/ahaidychuk/dependencyversionupdater/service/RepoServiceTest.java]: Yes, you're right, at this stage of development for this class it's redundant. Updated

**Review by yuriytkach — APPROVED** [review]

**Pull request merged**